### PR TITLE
Add information about data collection to docs

### PR DESCRIPTION
**Pull request by Bradamant3**
Adds list of data values collected and explains how to disable. See [community minutes from 11/10/2021](https://docs.google.com/document/d/1HgnK3wJIEg8uFlivdrhrPZYWTpElWWu3mhFDXj-bMWQ/edit#)

All we can do for now, related to but does not really address [this issue](https://github.com/kumahq/kuma/issues/3050)

PTAL @hhoover @austince 

N.B. Some values still need definition. Questions:

- product. Is this one of Kuma | Kong Mesh? (weird and unfortunate if so -- what else might it be?)
- unique_id. ID for the download? The install? A specific deployment of the same download/install?
- signal. No idea what this could be. Halp, plz
- cluster_id. My default assumption is an ID for a K8s cluster -- but how is it associated with the other values? Makes me wonder, just what is the parent for all these data points? (see Q about unique_id)
- Could use some clarification about DPs -- in all meshes apparently? (See also Q about cluster_id and parent for data points)

Also need instructions for universal. Who can help?

https://deploy-preview-582--kuma.netlify.app/docs/1.3.1/community/data-collection/

### CODE REVIEW

File: docs/docs/1.3.1/README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ The word "Kuma" means "bear" in Japanese (クマ).
 
 The core maintainer of Kuma is **Kong**, the maker of the popular open-source Kong Gateway 🦍.
 
+There's a new docs page that explains how your Kuma deployment collects data: what data is collected, and how to turn off data collection. See [data collection](/docs/1.3.1/community/data-collection).
```
Comment (lena-larionova) [inline docs/docs/1.3.1/README.md:17]:
```suggestion
There's a new docs page that explains how your Kuma deployment collects data. To find out what data is collected and how to turn off data collection, see [data collection](/docs/1.3.1/community/data-collection).
```
Slightly less awkward phrasing.

File: docs/docs/1.3.1/community/data-collection.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+| mode    | One of standalone or multizone |
+| hostname | The hostname of your service mesh |
+| signal |      | 
+| cluster_id |     |
+| dps_total | The total number of data plane proxies in your mesh | 
+| meshes_total | The total number of meshes deployed | 
+| zones_total | The total number of zones deployed | 
+| internal_services | Tne total number of services running inside your meshes | 
+| external_services | The total number of external services configured for your meshes |
+| services_ total | The total number of services in your mesh network |
```
Comment (lena-larionova) [inline docs/docs/1.3.1/community/data-collection.md:54]:
```suggestion
| services_total | The total number of services in your mesh network | 
```

**Comment by austince** [comment]
>     product. Is this one of Kuma | Kong Mesh? (weird and unfortunate if so -- what else might it be?)

This is specified in the build process. In kuma, it is `"Kuma"`, but could be changed by people with custom builds (e.g., Kong Mesh or another vendor). 

Source:
https://github.com/kumahq/kuma/blob/df0b18631833f7c9ed79f83b19262d391905ffe4/pkg/version/version.go#L9

>    unique_id. ID for the download? The install? A specific deployment of the same download/install?

A combination of the hostname and a random UUID, generated at runtime.

Source:
https://github.com/kumahq/kuma/blob/f07e2d488205d1d883cbf95f803b705f7727cf92/pkg/core/runtime/builder.go#L102 

>    signal. No idea what this could be. Halp, plz

Either "ping" or "start". "start" is sent once at boot, then "ping" for the lifetime of the deployment.

Source: https://github.com/kumahq/kuma/blob/8f1f7ff4d677cb9b1131e0088b7dd4d644a9924d/pkg/core/runtime/reports/reports.go#L200-L205

>    cluster_id. My default assumption is an ID for a K8s cluster -- but how is it associated with the other values? Makes me wonder, just what is the parent for all these data points? (see Q about unique_id)

I don't fully understand this one, but it appears like it is a random UUID that is persisted in the resource store to associate deployments across restarts. 
@jakubdyszkiewicz could probably explain more.

Source:
https://github.com/kumahq/kuma/blob/e83cfe5ac2359fe189bda8056db6fa7c1a21061d/pkg/clusterid/creator.go#L27-L42
https://github.com/kumahq/kuma/blob/54c21c3877a16f50e352ea45fa712cebc1ecac4f/pkg/clusterid/reader.go#L49-L59

>    Could use some clarification about DPs -- in all meshes apparently? (See also Q about cluster_id and parent for data points)

Looks to be across all meshes:
https://github.com/kumahq/kuma/blob/8f1f7ff4d677cb9b1131e0088b7dd4d644a9924d/pkg/core/runtime/reports/reports.go#L45-L52

File: docs/docs/1.3.1/community/data-collection.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+## What data is collected
+
+| Data field  | Definition   | 
+|---|---|---|---|---|
+| version  | The installed version of Kuma you're running | 
+| product  |   | 
+| unique_id  |   | 
+| backend  | Where your config is stored. One of in memory, etcd, Postgres | 
+| mode    | One of standalone or multizone |
+| hostname | The hostname of your service mesh |
```
Comment (austince) [inline docs/docs/1.3.1/community/data-collection.md:46]:
```suggestion
| hostname | The hostname of each of your Kuma Control Planes |
```
Or something similar?

**Review by austince — APPROVED** [review]
Thanks so much for taking this on @Bradamant3 ! I just had a couple of comments.

File: docs/docs/1.3.1/community/data-collection.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  --env-var KUMA_REPORTS_ENABLED=false
+```
+
+:::
+::: tab "Helm"
+
+Set the environment variable:
+
+```shell
+helm install --version 0.7.1 --namespace kuma-system \
```
Comment (lahabana) [inline docs/docs/1.3.1/community/data-collection.md:29]: the version is unecessary

**Review by lahabana — APPROVED** [review]
LGTM with @austince 's suggestions

File: docs/docs/1.3.1/community/data-collection.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  --set controlPlane.envVars.KUMA_REPORTS_ENABLED=false \
+   kuma kuma/kuma
+```
+
+:::
+::::
```
Comment (jakubdyszkiewicz) [inline docs/docs/1.3.1/community/data-collection.md:35]: I created `configuration` section for this specific reason. I noticed that we are describing everytime how to configure Kuma. I think what we should do from now one is to say that you need to set `KUMA_REPORTS_ENABLED=false` and point user to configuration for them to learn how to set it

File: docs/docs/1.3.1/community/data-collection.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+## What data is collected
+
+| Data field  | Definition   | 
+|---|---|---|---|---|
+| version  | The installed version of Kuma you're running | 
+| product  |   | 
+| unique_id  |   |
```
Comment (jakubdyszkiewicz) [inline docs/docs/1.3.1/community/data-collection.md:43]: ID of an instance of Kuma Control Plane. It's different every time when you restart CP instance.

File: docs/docs/1.3.1/community/data-collection.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+| Data field  | Definition   | 
+|---|---|---|---|---|
+| version  | The installed version of Kuma you're running | 
+| product  |   | 
+| unique_id  |   | 
+| backend  | Where your config is stored. One of in memory, etcd, Postgres | 
+| mode    | One of standalone or multizone |
+| hostname | The hostname of your service mesh |
+| signal |      |
```
Comment (jakubdyszkiewicz) [inline docs/docs/1.3.1/community/data-collection.md:47]: either `start` or `ping`. We send `start` when CP starts and then `ping` every 1 hour

File: docs/docs/1.3.1/community/data-collection.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+| Data field  | Definition   | 
+|---|---|---|---|---|
+| version  | The installed version of Kuma you're running | 
+| product  |   | 
+| unique_id  |   | 
+| backend  | Where your config is stored. One of in memory, etcd, Postgres | 
+| mode    | One of standalone or multizone |
+| hostname | The hostname of your service mesh |
+| signal |      | 
+| cluster_id |     |
```
Comment (jakubdyszkiewicz) [inline docs/docs/1.3.1/community/data-collection.md:48]: Unique identifier for the whole Kuma cluster. The value is the same between all CP instances. On multizone deployment, it's the same for both Global and Zone CP.

File: docs/docs/1.3.1/community/data-collection.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+:::
+::::
+
+## What data is collected
+
+| Data field  | Definition   | 
+|---|---|---|---|---|
+| version  | The installed version of Kuma you're running | 
+| product  |   |
```
Comment (jakubdyszkiewicz) [inline docs/docs/1.3.1/community/data-collection.md:42]: The name of the product. Static value of "Kuma".

**Pull request merged**